ironfish/src/rpc/routes/chain/getBlocks.ts

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { Assert } from '../../../assert'
+import { getBlockSize, getTransactionSize } from '../../../network/utils/serializers'
+import { FullNode } from '../../../node'
+import { GENESIS_BLOCK_SEQUENCE } from '../../../primitives/block'
+import { BufferUtils } from '../../../utils'
+import { RpcNotFoundError, RpcValidationError } from '../../adapters'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+import { RpcBlock, RpcBlockSchema, serializeRpcBlockHeader } from '../types'
+import { RpcTransaction } from './types'
+
+const MAX_BLOCKS_RANGE = 30
```

Comment:
Marking this to think about a bit more, but no action needed yet.

After looking around the RPC code, I don't think we have max limits for RPC calls anywhere else yet, so I want to think a bit about what our pattern should be.

Reply:
I think it's ok if we remove it, since you are the only one that can hit your own RPC anyway.

No one else can DDOS your node.